[4mRunning "env:test" (env) task[24m

[4mRunning "mochaTest:kmp_cache_instance" (mochaTest) task[24m

[0m[0m
[0m  Instance Model[0m
[0m    set: when attempting to cache an instance[0m
    [32m  âœ“[0m[90m Test 019: cache_instance returns an 400 error when instanceId is missing[0m

[0m  Instance Model[0m
[0m    set: when attempting to cache an instance[0m
    [32m  âœ“[0m[90m Test 020: cache_instance returns an 400 error when openRosaId is missing[0m

[0m  Instance Model[0m
[0m    set: when attempting to cache an instance[0m
    [32m  âœ“[0m[90m Test 021: cache_instance returns an 400 error when openRosaServer is missing[0m

[0m  Instance Model[0m
[0m    set: when attempting to cache an instance[0m
    [32m  âœ“[0m[90m Test 022: cache_instance returns an 400 error when instance is missing[0m

[0m  Instance Model[0m
[0m    set: when attempting to cache an instance[0m
    [31m  1) Test 023(fault): cache_instance returns the survey object when the instance is cached successfully[0m

[0m  Instance Model[0m
[0m    set: when attempting to cache an instance[0m
    [31m  2) Test 024: get_cached_instance returns an 400 error when instanceId is missing[0m


[92m [0m[32m 4 passing[0m[90m (142ms)[0m
[31m  2 failing[0m

[0m  1) Instance Model set: when attempting to cache an instance Test 023(fault): cache_instance returns the survey object when the instance is cached successfully:
[0m[31m     Error: Bad request. Survey information not complete or invalid[0m[90m
    at app/models/instance-model.js:25:21
    at tryToUnwrap (node_modules/lie/lib/index.js:171:5)
    at tryCatch (node_modules/lie/lib/index.js:183:17)
    at safelyResolveThenable (node_modules/lie/lib/index.js:174:16)
    at new Promise (node_modules/lie/lib/index.js:32:5)
    at Object._cacheInstance [as set] (app/models/instance-model.js:23:12)
    at Context.<anonymous> (test/server/cache_instance-18.spec.js:43:34)
  
[0m
[0m  2) Instance Model set: when attempting to cache an instance Test 024: get_cached_instance returns an 400 error when instanceId is missing:

      [31mAssertionError: expected 400 to equal 404[0m
      [32m+ expected[0m [31m- actual[0m

      [31m-400[0m
      [32m+404[0m
      [0m[90m
    at getBasePromise.then.then.newArgs (node_modules/chai-as-promised/lib/chai-as-promised.js:302:22)
    at Array.<anonymous> (node_modules/lie/lib/index.js:88:21)
    at nextTick (node_modules/immediate/lib/index.js:61:18)
    at _combinedTickCallback (internal/process/next_tick.js:73:7)
    at process._tickDomainCallback (internal/process/next_tick.js:128:9)
  
[0m


[[90mmochawesome[39m] Report JSON saved to /app/mochawesome-report/output-1511917754.json

[[90mmochawesome[39m] Report HTML saved to /app/mochawesome-report/output-1511917754.html

[33mWarning: Task "mochaTest:kmp_cache_instance" failed. Use --force to continue.[39m

[31mAborted due to warnings.[39m


[4mExecution Time[24m[90m (2017-11-29 01:09:10 UTC-0)[39m
loading tasks                 [34m3.9s[39m  [34mâ–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡â–‡ 78%[39m
mochaTest:kmp_cache_instance  [34m1.1s[39m  [34mâ–‡â–‡â–‡â–‡â–‡â–‡â–‡ 22%[39m
[35mTotal 5s[39m

